lib/mayaUsd/ufe/UsdUndoSetKindCommand.cpp

```diff
@@ -17,9 +17,7 @@
 
 #include <mayaUsd/undo/UsdUndoBlock.h>
 
-#include <pxr/base/tf/token.h>
 #include <pxr/usd/usd/modelAPI.h>
-#include <pxr/usd/usd/prim.h>
```

Comment:
What defines "unused" here? `TfToken` and `UsdPrim` are most definitely used below.

Reply:
The includes for TfToken and UsdPrim are already in the header.  The ones in the *.cpp are unnecessary.

https://github.com/Autodesk/maya-usd/blob/1fd0c0f3f8b643456b7fe808f4fdefd06d734c43/lib/mayaUsd/ufe/UsdUndoSetKindCommand.h#L23